client/src/hooks/queries/useGetInteractions.ts

```diff
@@ -63,6 +63,13 @@ const getInteractionsByDrugsQuery = gql`
             type
             directionality
           }
+          publications {
+            pmid
+          }
+          sources {
+            id
```

Comment:
We shouldn't need to pull ID for anything on the frontend -- this can be removed